test/tests/docker-build.sh

```diff
@@ -30,4 +30,4 @@ fi
 
 cp -RL "$dir" "$tmp/dir"
 
-command docker build -t "$imageTag" "$tmp" > /dev/null
+error=$(command docker build -t "$imageTag" "$tmp" 2>&1) || { echo -e "$error\n" 1>&2; exit 1; }
```

Comment:
What issue is this solving? It is not clear to me why you need the `-e` and `\n`, since `echo` adds a newline automatically.

Reply:
This changes supresses the Docker build output unless there is an error to get a cleaner view of the test results

Reply:
```suggestion
error=$(command docker build -t "$imageTag" "$tmp" &>) || { echo "$error" >&2; exit 1; }
```

Reply:
Might not the docker build output be useful otherwise? Typically it is good to err on the side of verbosity in CI builds.

Reply:
Having the Docker output there makes it very hard to see which tests fail. The test Dockerfile themselves are usually extremely simple.